in_toto/verifylib.py

```diff
@@ -218,7 +219,7 @@ def verify_all_steps_signatures(layout, links_dict):
   <Arguments>
```

Comment:
I know, but you changed the implementation of the function to verify signatures of potentially multiple Links for one step. That's why I suggest to also change the docstring.